pipelines/qc/README.md

```diff
@@ -0,0 +1,138 @@
+# QC pipeline guide
+
+## Overview
+
+This pipeline takes a sampleset run through the vo-agam pipeline, and runs scripts to estimate contamination and to calculate some alignment summary stats.
+As part of this it applies filters which blacklist failing samples and makes a sex call based on the ratio of coverage of 3L:X.
+The output is a `tsv` file that contains the pertinent statistics for each sample, as well as filters to be applied.
+
+Filter definitions are specified in `config.yml`
+
+## Running the pipeline
+
+### 1. Ensure the data are available in the relevant environment. In this case the Oxford cluster.
+
+The input to this pipeline are a set of zipped zarr files, in the standard `calldata/GT` etc. format.
+
+In the case of *Anopheles gambiae* these are copied from the Cambridge cluster to a location on the Oxford cluster nightly. 
+
+An rsync command may look something like this:
+
+`rsync -options othercluster.ac.uk:/path/on/other/cluster/ /path/to/inputdir`
+
+The command used for the AG1000G project (to foxtrot from malsrv2) is:
+
+```
+# first set up a port forward to local host to navigate two-step Sanger system
+ssh -N -f -L localhost:2222:malsrv2:22 nh7@ssh.sanger.ac.uk
+
+# remove the "n" flag to take off dry-run mode
+rsync --progress -nrogptvLe "ssh -p 2222" \
+  nh7@localhost:/lustre/scratch118/malaria/team112/pipelines/setups/vo_agam/symlink/vo_agam_vcf2zarr/convert_vcf_to_zarr_serial_contig/ \
+  /kwiat/vector/mirror/observatory/vo_agam/genotype_zarrs/
+```
+
+### 2. Ensure your repository contains `vector-tools` as a submodule. 
+
+The second thing to check is that the `vector-tools` repository has been added as a submodule of your repository. In this example case, the repository for the AG1000G project is `vector-ops`. All commands run here are done so from the root directory of the `vector-ops` repository.
+
+### 3. Create the configfile from the example file provided.
+
+In `vector-tools/pipelines/qc` the `config.yml` file is that used by the AG1000G project. Copy this file into an appropriate place in your repo (for `vector-ops` we used `pipelines/qc/config.yml`, and edit the filepaths and filter settings where necessary. 
+
+### 4. Run the pipeline!
+
+To run the pipeline (e.g. on the Uganda sampleset) invoke snakemake using the `SGE` system on the Oxford cluster (from `vector-ops`):
+
+```
+bash vector-tools/pipelines/qc/submit.sh qc_report \
+  --config fofn=${pwd}/tracking/AG1000G-UG/agam.fofn.tsv \
+  --configfile vector-tools/pipelines/qc/config.yml \
+  -d /kwiat/vector/observatory/production/vo_agam/qc
+```
```

Comment:
Another part of the story is that we do not leave any trace of actually running the pipeline on specific dataset in github.

Reply:
Yes- that's the intention